.github/pull_request_template.md

```diff
@@ -23,8 +23,6 @@ Before submitting this PR, please make sure:
 - [ ] You have added an entry to the relevant `CHANGELOG.md` (see
       ["keep a changelog"] for more information).
 - [ ] Your commit messages follow the [conventional commit] style.
-- [ ] Your changes are squashed into a single commit (unless there is a _really_
```

Comment:
Are we removing this since we default to squash merges for the PRs?

If so, we should probably remove the other one towards the end of the file.

Reply:
Yes, that is why. Me and Andrew T talked it over and thought squash merges accomplish the same thing (mostly) and are a whole lot easier for maintainers (us and external ones) to keep up with.

Good catch on those other 2 lines I missed.